nifi-provenance-reporting-tasks/src/main/java/io/egm/nifi/reporting/EmailProvenanceReporter.java

```diff
@@ -148,6 +149,17 @@ public class EmailProvenanceReporter extends AbstractProvenanceReporter {
             .required(false)
             .addValidator(StandardValidators.NON_EMPTY_VALIDATOR)
             .build();
+    public static final PropertyDescriptor GROUP_SIMILAR_ERRORS = new PropertyDescriptor.Builder()
+            .name("Group Similar Errors")
+            .displayName("Group Similar Errors")
+            .description("Specifies whether to group multiple error events into a single email or not." +
+                    " Set to true to receive an email with grouped errors. " +
+                    "Set to false to receive individual emails for each error." +
+                    " The grouping is by process and error information")
```

Comment:
I still don't know what is a process (wrt NiFi)

Reply:
can you clarify more ?

Reply:
In NiFi, there are processors, process groups. But nothing called "process".